src/components/shared/formItems/ProjectPayoutMods.tsx

```diff
@@ -419,14 +417,13 @@ export default function ProjectPayoutMods({
             />
           ) : (
             <FormItems.ProjectHandle
-              name="project"
+              name="handle"
               requireState="exists"
               formItemProps={{
                 label: 'Project handle',
                 rules: [{ required: true }],
               }}
-              value={form.getFieldValue('handle')}
```

Comment:
By setting the `name` correctly above, we don't need to manually retrieve and set the form field 🙂 .